### PR TITLE
Update muon manual testing  docs

### LINKED ISSUE #33904: Muon docs update
This is a list of changes needed to the muon docs for testing from https://github.com/mantidproject/mantid/issues/33834

**General**
Add link to feature flag doc

**ALC**
Change from marker to add section

**MUSR**
plot selector 
list the wrong globals
Change "select data" to "display parameters for"

**PSI**
Change from 0.007 to 0.006

**EMU**
Move the line about external plots
Remove bit on stepping through the plots in superconducting 

**FDA**
The default FFT settings have changed
Phasequad is added in a different way
Maxent output names need updating

**HIFI**
Change to selected groups

### PR DESCRIPTION
**Pull request by MohamedAlmaki**
**Description of work.**
The PR updates different sections in the muon manual testing docs.

**To test:**

Fixes #33904. 

*This does not require release notes* because **it is a dev doc update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
I think the best test for this is to use it for the mob testing.

**Comment by AnthonyLim23** [comment]
changes from https://github.com/mantidproject/mantid/issues/34946

An instruction to explain feature flags is needed since they are necessary for one of the tests
We needed to change the fit peak centre for the last step of the super-conducting copper test for the fit to be sensible
On the HIFI Transverse Field Simultaneous Fitting test, our first fit had Chi > 100 but still displayed as a success (I think this is fine). Also it should say Chi Squared rather than just Chi in the testing doc
In the Transverse Field Asymmetry test, TF Asymmetry Mode is in a combo box, not a check box in the bottom table as described

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_EMU.rst [hunk 1]
```diff
@@ -94,6 +94,7 @@ Superconducting Copper Test
 	- Set the ``Select Data`` to ``field_danfysik`` and ``Lambda``
 	- The plot will look similar to the one below
 	- Add a peak function (e.g. Gaussian)
+	- Move the peak centre to around 0.75
```
Comment (AnthonyLim23) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_EMU.rst:97]: did you mean 75?

Comment (MohamedAlmaki) [reply dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_EMU.rst:97]: Yes

**Review by AnthonyLim23 — APPROVED** [review]

**Review by DannyHindson — APPROVED** [review]

**Pull request merged**